From 369046ca20c1a7eaa93c7c94b5e4a8a94292412b Mon Sep 17 00:00:00 2001
From: Alex Chermenin <alex@chermenin.ru>
Date: Sun, 11 May 2025 21:53:25 +0500
Subject: [PATCH] debug-shell: configure environment, unhide cursor

---
 units/debug-shell.service.in | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/units/debug-shell.service.in b/units/debug-shell.service.in
index 2c2c8dd4f5..2002742e47 100644
--- a/units/debug-shell.service.in
+++ b/units/debug-shell.service.in
@@ -16,8 +16,8 @@ ConditionPathExists={{DEBUGTTY}}
 After=systemd-vconsole-setup.service
 
 [Service]
-Environment=TERM=linux
-ExecStart={{SUSHELL}}
+Environment=ENV=/etc/profile
+ExecStart=/bin/sh -c 'echo -en "\033[?25h"; exec /bin/sh'
 Restart=always
 RestartSec=0
 StandardInput=tty
-- 
2.48.1

